aip/cloud/25146.md

```diff
@@ -0,0 +1,34 @@
+---
+id: 25146
+state: draft
+created: TBD
+placement:
+  category: gcp
+---
+
+# Generic Fields
+
+Guidelines related to generic fields in [AIP-146](../0146.md) **should** be applied
+with the following exceptions.
+
+## Maps
+
+The value type of a map **must not** be of type Message
+
+The rationale is:
+- Cloud APIs are generally used through JSON. In JSON keys are generally field names
+rather than arbitrary strings or references to other resources. Many tools that interact
+with the JSON API will not be able to handle dynamic keys with static Message values.
+
+### Alternative
+
+Consider using a separate resource rather than a map of objects.
+
+**or**
```

Comment:
I'm not a fan of this dramatic line break and solo "**or**" - can we either combine these into a few sentences or make a (unbounded*) bulleted list?

*unbounded: meaning "here are some suggested alternatives, but there may be others"